plugins/woocommerce/tests/legacy/unit-tests/woocommerce-admin/api/reports-categories.php

```diff
@@ -187,7 +187,6 @@ public function test_get_reports_categories_sort() {
 		$order->save();
 
 		WC_Helper_Queue::run_all_pending();
-		CategoryLookup::instance()->regenerate();
```

Comment:
Is it intended to remove this line?

Reply:
Yes, it's causing other tests to fail. This isn't really needed for this test to pass.